### DISCUSSION TITLE
Manual sorting in M2M interface table layout is not obvious

### DISCUSSION THREAD

**Discussion by hanneskuettner** [directus/directus#24907] [Feature Requests]
EDIT: Turns out it is not a bug, but a well hidden feature. The sorting does work, but requires the user to click on a toggle that has different meanings throughout Directus.

### Describe the Bug

The sorting functionality does not work in the M2M interface if the selected layout is `Table`, see attached video.

First, I show that sorting does work for the relation in the `List` layout, then that it is broken for `Table`.

https://github.com/directus/directus/assets/4376726/e7375170-4e45-4151-908c-d097a3ac53c0

Notice how the cursor does not change when hovering over the drag handle.

### To Reproduce

- Create a sortable M2M relation
- Change layout to `Table` and try sorting the items

### Directus Version

v10.11.3

### Hosting Strategy

Self-Hosted (Docker Image)

**Comment by paescuj** [comment]
You need to enable sorting by clicking on the "sort" icon:

https://github.com/directus/directus/assets/5363448/75976c82-fed6-401e-a722-e2ceb04c80b6

But I agree that this is not super intuitive... Wanna keep the issue open to discuss potential improvements?

**Comment by hanneskuettner** [comment]
Wow. Totally missed that. Thanks for pointing that out, yes, let's keep it open, I'll change the title and label.

**Comment by poclerson** [comment]
Could we have the option the toggle manual sorting on by default?
